underfs/stocator/src/main/java/alluxio/underfs/stocator/StocatorUnderFileSystem.java

```
@@ -0,0 +1,323 @@
+    }
+  }
+
+  @Override
+  public void close() throws IOException {
+    LOG.debug("close");
+    mFileSystem.close();
+  }
+
+  @Override
+  public void connectFromMaster(String hostname) {
+    LOG.debug("connect from master");
+  }
+
+  @Override
+  public void connectFromWorker(String hostname) {
+    LOG.debug("connect from worker");
+  }
+
+  @Override
+  public FSDataOutputStream create(String path, CreateOptions options)
+      throws IOException {
+    if (PathUtils.isTemporaryFileName(path)) {
+      path = PathUtils.getPermanentFileName(path);
+      LOG.debug("Modified path: {}", path);
+    }
+    try {
+      return FileSystem.create(mFileSystem, new Path(path), null);
+    }  catch (IOException e) {
+      LOG.error("Failed to create {}", path);
+      throw e;
+    }
+  }
+
+  @Override
+  public boolean deleteDirectory(String path, DeleteOptions options) throws IOException {
+    return isDirectory(path) && delete(path, options.isRecursive());
+  }
+
+  @Override
+  public boolean deleteFile(String path) throws IOException {
+    return isFile(path) && delete(path, false);
+  }
+
+  @Override
+  public boolean exists(String path) throws IOException {
+    LOG.debug("Exists {}", path);
+    return mFileSystem.exists(new Path(path));
+  }
+
+  /**
+   * Gets the block size in bytes. There is no concept of a block in Swift and the maximum size of
+   * one file is 4 GB. This method defaults to the default user block size in Alluxio.
+   *
+   * @param path the path to the object
+   * @return the default Alluxio user block size
+   * @throws IOException this implementation will not throw this exception, but subclasses may
+   */
+  @Override
+  public long getBlockSizeByte(String path) throws IOException {
+    return Configuration.getBytes(PropertyKey.USER_BLOCK_SIZE_BYTES_DEFAULT);
+  }
+
+  @Override
+  public Object getConf() {
+    LOG.debug("getConf is not supported when using StocatorUnderFileSystem, returning null.");
```

instead of leaving debug message and return null, can we throw `UnsupportedOperationException` instead? Same comment applied to the following few methods that `LOG.debug` and return null